### DISCUSSION TITLE
Include StylusTip Calibration

### DISCUSSION THREAD

**Discussion by LeidyMoraV** [PlusToolkit/PlusLib#1180] [Q&A] [answered]
Hi there! I'm currently working on a tracked ultrasound project. I've successfully reconstructed the volume, and when I directly touch the Stylus Marker to the spine phantom, the position matches what I see in 3D slicer. However, when I place the marker on the Stylus Phantom, it seems like there's a slight discrepancy between the marker and the point I touch. I suspect this issue is related to the relationship between the Stylus Marker and the Stylus Tip. I've performed calibration to obtain the corresponding matrix, but I'm unsure if I've properly included it in the configuration file. Can anyone advise on how to ensure the calibration matrix is correctly applied in the configuration file?

config file: https://pennstateoffice365-my.sharepoint.com/:u:/g/personal/lfm5705_psu_edu/EVizQgTZTPtAu9aJ-s3-FNcBifRF76UjkeOFDexwK_sAfQ?e=FfucDN

**Comment by Sunderlandkyl** [comment]
If the discrepancy is in 3D Slicer, can you share your scene?

It doesn't look like your are sending the PhantomToReference transform, which would probably be needed to visualize the phantom at the correct location in Slicer.

**Reply by LeidyMoraV** [reply]
Scene File: https://pennstateoffice365-my.sharepoint.com/:f:/g/personal/lfm5705_psu_edu/EqmZqKnS1ytLq73oRsh2fRYB6NqpL5Ih75Hklg_l1LCzbw?e=inf8bI

**Reply by Sunderlandkyl** [reply, marked as answer]
You are currently only sending StylusToReference. If you want the stylus to appear in the correct location in Slicer, using the stylus calibration in the config file, you should send and use StylusTipToReference instead.

Edit :
>        <Transform Name="StylusToReference" />

To:
>        <Transform Name="StylusTipToReference" />
